app/authentication_views.py

```diff
@@ -17,12 +16,12 @@ def register():
         username = request.form['username']
         email = request.form['email']
         password = request.form['password']
-        birthDay = request.form['birthDay'].split("-")[0]
+        birthday = request.form['birthDay'].split("-")[0]
```

Comment:
what if birthday is empty or it has no -? 
validate or add todo for validation

Reply:
I get in that format from html's form, I will put some validation for it and change it for a full date not only a year

Reply:
everything has to be validated on backend side, you cant rely on frontend for validation